charts/aws-ebs-csi-driver/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "0.9.0"
 name: aws-ebs-csi-driver
 description: A Helm chart for AWS EBS CSI Driver
-version: 0.9.6
+version: 0.9.7
```

Comment:
Can we update this to 0.9.8? We recently merged another change.

Reply:
done